stripe/src/main/java/com/stripe/android/PaymentSessionConfig.kt

```diff
@@ -92,6 +94,7 @@ data class PaymentSessionConfig internal constructor(
         private var allowedShippingCountryCodes: Set<String> = emptySet()
         private var shippingInformationValidator: ShippingInformationValidator? = null
         private var shippingMethodsFactory: ShippingMethodsFactory? = null
+        private var windowFlags: Int? = null
```

Comment:
why is this `Int` instead of `WindowManager.LayoutParams`? It looks like some of the values can be floats, e.g. `BRIGHTNESS_OVERRIDE_FULL`

Reply:
@csabol-stripe the argument to this method should only be flags (i.e. constants that start with `FLAG_` from https://developer.android.com/reference/android/view/WindowManager.LayoutParams). It can be a single flag, or bitwise or-ed flag.